services/payment-service/actions/payment/payment.action.ts

```diff
@@ -1,17 +1,19 @@
 import { PaymentRepository } from './../../repository/payment.repository';
 import { PaymentCreatePayload } from "../../types/payment.interface";
 import Stripe from 'stripe';
-import { envMode, serviceConfig } from '../../configs/config';
-
-const config = serviceConfig[envMode]
+import { serviceConfig } from '../../configs/config';
 
 export class PaymentActions {
     private paymentRepository: PaymentRepository;
     private stripe: Stripe;
 
     constructor() {
         this.paymentRepository = new PaymentRepository();
-        this.stripe = new Stripe(config.stripe.secretKey)
+        this.stripe = new Stripe(serviceConfig.stripe.secretKey || '')
+    }
+
+    async getPayments() {
```

Comment:
Everything inside /actions, please follow the standard in user service. We want to simplify and standardize the code, and please add swagger.